traffic_portal/app/src/common/modules/form/cdn/form.cdn.tpl.html

```diff
@@ -41,6 +41,9 @@
                     <span class="caret"></span>
                 </button>
                 <ul class="dropdown-menu-right dropdown-menu" uib-dropdown-menu>
+                    <li ng-if="!cdn.notificationCreatedBy" role="menuitem"><a ng-click="toggleNotification(cdn)">Create CDN Notification</a></li>
+                    <li ng-if="cdn.notificationCreatedBy" role="menuitem"><a ng-click="toggleNotification(cdn)">Delete CDN Notification</a></li>
```

Comment:
Anchor tags should be links - these aren't performing navigation, so they ought to be `button[type="button"]`s

Reply:
removing. these were supposed to be removed but i forgot about them.